torch/csrc/jit/passes/subgraph_rewrite.cpp

```diff
@@ -44,6 +43,39 @@ void SubgraphRewriter::runOnGraph(std::shared_ptr<Graph>& graph) {
   }
 }
 
+/**
+ * Clone \p SUBGRAPH_TO_COPY inserting it after \p ANCHOR and using \p INPUTS as
+ * incoming values for the new subgraph. Return a vector of values produced by
+ * the inserted subgraph.
+ */
+static std::vector<Value*> insertSubgraphCloneAfter(
```

Comment:
This appears to be an implementation of ir.h's `inlineCallTo` which is the preferred way of inlining a call of one graph into another.